extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
Yes, possibly -- I'm not sure offhand, but that way the deps are still dynamically loaded based on whether there's an interaction that actually uses them. In Pranav's version I think they were being hardcoded specifically in the frontend.